docs/json.md

```diff
@@ -483,8 +483,8 @@ it explains, among other things, how custom serializers are bound to classes.
 Transformation capabilities are provided by the abstract [JsonTransformingSerializer] class which implements [KSerializer]. 
 Instead of direct interaction with `Encoder` or `Decoder`, this class asks you to supply transformations for JSON tree 
 represented by the [JsonElement] class using the 
-[transformSerialize][JsonTransformingSerializer.transformSerialize] and 
-[transformDeserialize][JsonTransformingSerializer.transformDeserialize] methods. Let us take a look at the examples.
+`transformSerialize` and 
```

Comment:
why can't we reference it here?

Reply:
It's a Dokka bug: https://github.com/Kotlin/dokka/issues/1522